kiwi/system/uri.py

```diff
@@ -134,13 +141,13 @@ def credentials_file_name(self):
 
         :rtype: str
         """
-        uri = urlparse(self.uri)
+        uri = self._has_credentials_query()
```

Comment:
nitpick, but I think this variable name is confusing

Reply:
What about `credentials` instead of `uri`?

Reply:
ok, yes this is a confusing name, I'll change it